README.md

```diff
@@ -22,8 +22,7 @@ sudo mv /tmp/eksctl /usr/local/bin
 Alternatively, macOS users can use [Homebrew](https://brew.sh):
 
 ```
-brew tap weaveworks/tap
-brew install weaveworks/tap/eksctl
+brew install eksctl
```

Comment:
just curious if the RC will also available ?

Reply:
Unstable versions can be made available via `head` (see https://docs.brew.sh/Formula-Cookbook#unstable-versions-head) but I will ask the staff what the best way is to make the RC release available.
Right now `eksctl`'s RC versions are not available via homebrew, right?

Reply:
I think it is as per PR https://github.com/weaveworks/eksctl/pull/2033 if I am not wrong.

Reply:
Thank you. I will just add the same line when I update the formula the next time ok @sayboras ?

Reply:
I am ok, just want to highlight this, so that we can have a similar experience. Not sure about other users, I sometimes use RC to test it out with new feature.

Keen to hear inputs from maintainers as well.